Formula/cbonsai.rb

```diff
@@ -0,0 +1,20 @@
+class Cbonsai < Formula
+  desc "Console Bonsai is a bonsai tree generator, written in C using ncurses"
+  homepage "https://github.com/mhzawadi/homebrew-cbonsai"
+  url "https://github.com/mhzawadi/homebrew-cbonsai/archive/refs/tags/v1.3.4.tar.gz"
+  sha256 "d984f50afab06c5160b7f8577672397203d9d52f6857648c56f0fca4a409fb51"
+  license "GPL-3.0"
+  depends_on "ncurses"
+  depends_on "pkg-config"
+
+  def install
+    ENV["PKG_CONFIG_PATH"] = "$PKG_CONFIG_PATH:/usr/local/Cellar/ncurses/6.4/lib/pkgconfig/"
+    ENV.append_path "PKG_CONFIG_PATH", "/opt/homebrew/Cellar/ncurses/6.4/lib/pkgconfig/"
+    system "make"
+    bin.install "cbonsai"
+  end
+
+  test do
+    system bin/"cbonsai", "-p"
```

Comment:
Can we ensure some output here?

Reply:
umm, would the help be a better output as that can be tested.

as the output from the `-p` option is a tree that will change each time it runs
![image](https://github.com/Homebrew/homebrew-core/assets/11944327/e2d76204-daf0-477a-9ca4-7096d1abc3ed)

Reply:
There are probably flags to make the tree fixed